frontend/src/components/common/form/InputGroup.tsx

```diff
@@ -74,7 +74,8 @@ export const InputGroup: React.FC<InputGroupProps> = ({
         disabled ? 'disabled' : '',
       )}
     >
-      {!!label && <Form.Label>{label}</Form.Label>}
+      {!!label && !required && <Label>{label}</Label>}
+      {!!label && required && <Label required>{label}</Label>}
```

Comment:
These 2 can be replaced with `{!!label && <Label required={required}>{label}</Label>}`